src/tests/ftest/launch.py

```diff
@@ -925,50 +939,75 @@ def archive_config_files(avocado_logs_dir):
     archive_files(
         destination, host_list, "{}/*_*_*.yaml".format(configs_dir))
 
-
-def archive_files(destination, host_list, source_files):
+def archive_files(destination, host_list, source_files, do_tar=False):
```

Comment:
As mentioned above, I believe a cleaner and more consistent approach would be to revert the changes to the `archive_files` method, use `*.log` for cart logs, and allow CI to do the sized-based compression.

Reply:
We'll remove the offending changes here, and plan to fix the log arching in [this PR](https://jira.hpdd.intel.com/browse/DAOS-5662).